ocfweb/reservations/templates/reservations/index.html

```diff
@@ -0,0 +1,92 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-8 ocf-content-block">
+
+            <form method="post" action="{% url 'request_reservation' %}">
+                {% csrf_token %}
+                <p>
+                    This form is <strong>only</strong> for student groups desiring to
+                    use the OCF's lab space. If you want to create an individual account
+                    please visit <a href="{% url 'register' %}">this link</a>. If you
+                    want to request an account for a student group, please visit
+                    a staff member in the OCF lab during
```

Comment:
fix copy, this refers to creating accounts

Reply:
I intended the wording here to be exactly as is.

Reply:
honestly don't understand what you're saying -- it seems like you deleted all this anyway?

Reply:
Could you clarify? I don't see anything deleted in this file

Reply:
looks like it was just renamed. i'll re-open my comments on the new file.